### PR TITLE
add more installation details

### PR DESCRIPTION
**Pull request by pravipati**
@swinton thanks for making this 😸. I added some documentation related to our conversation this morning.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,14 +19,18 @@ I like using a REPL to experiment with ideas and get comfortable with an API bef
 npm install -D github-app-repl
 ```
 
+It expects a `APP_ID` environment variable, and corresponding `.pem` file. Most Probot projects should have this.
```
Comment (swinton) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -19,14 +19,18 @@ I like using a REPL to experiment with ideas and get comfortable with an API bef
 npm install -D github-app-repl
 ```
 
+It expects a `APP_ID` environment variable, and corresponding `.pem` file. Most Probot projects should have this. 
+
```
Comment (swinton) [inline README.md:22]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -19,14 +19,18 @@ I like using a REPL to experiment with ideas and get comfortable with an API bef
...
+To find these details for a GitHub App you have installed, log into GitHub and from your profile view go to `Settings > Developer Settings > GitHub Apps > YOUR_APP > EDIT`
```
Comment (swinton) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -19,14 +19,18 @@ I like using a REPL to experiment with ideas and get comfortable with an API bef
...
+
+From here, you can generate a new private key if you'd like. You will need either need to move the `.pem` file into `gitHub-app-repl`'s directory or update the `.env` file to include a file path to your key.
```
Comment (swinton) [inline README.md]:
```suggestion
## Configuration

A couple of environment variables are required for configuration:

1. **`APP_ID`**: This should be set to the ID of your GitHub App. 
    - This is available from the settings page of your GitHub App, as `App ID`.
1. **`PRIVATE_KEY_PATH`**: This should be set to a  local filesystem path to a private key for your GitHub App. 
    - More information on generating and downloading private keys for your GitHub Apps is [available in the documentation](https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#generating-a-private-key).

These environment variables can either be set directly, or via a `.env` file that follows [this example](https://github.com/swinton/github-app-repl/blob/master/.env.example).
```

**Review by swinton — CHANGES_REQUESTED** [review]
Thanks @pravipati. I can see there's some friction in setting this up, and I agree it would be good to improve it.

So that the README doesn't become out-of-date, and so that the instructions also apply to Organization-owned GitHub Apps, I think we should link to the existing documentation where possible, such as [Generating a private key](https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#generating-a-private-key).

I've made a few suggestions along these lines :bow:

**Comment by pravipati** [comment]
@swinton agreed! just batched and applied your suggestions 👏

**Review by swinton — APPROVED** [review]
Thanks! 🚀

**Pull request merged**